app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,39 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
```

Comment:
No no no ... Add missing objects to the model itself inside the `model` method and loop only through the model